app/views/user_mailer/challenge_assignment_notification.html.erb

```diff
@@ -1,13 +1,11 @@
 <% content_for :message do %>
-  <p>
-    You have been assigned the following request in the <%= style_link(@collection.title, collection_url(@collection)) %> challenge at the Archive of Our Own!
-  </p>
+  <p><%= t ".html.part1", link: style_link(@collection.title, collection_url(@collection)) %></p>
```

Comment:
I took a quick look at this PR in my test logs, and this link appears to give exposed HTML: `You have been assigned the following request in the &lt;a href=&quot;http://archiveofourown.org/collections/Awesome_Gift_Exchange&quot; style=&quot;color:#990000&quot;&gt;Awesome Gift Exchange&lt;/a&gt; challenge at the Archive of Our Own!`